packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountSelection/TransferFormAccountSelection.scss

```diff
@@ -33,6 +33,8 @@
             padding: 0 1.6rem 1.6rem;
             gap: 1.6rem;
             overflow-y: unset;
+            //prevent content bouncing on scroll on iPhone devices
+            overscroll-behavior: none;
```

Comment:
So this disables some default expected browser behavrious, to fix the problem, which looks like problem with our header. Probably would be better to fix header or make header animation faster or even remove the header animation whatsover (I think just making it faster should prevent it from "resonating" with rubber-band scrolling effect on ios)

Reply:
after discussion with @wojciech-deriv, we decided to leave this solution

Reply:
had a chat with @george-usynin-deriv, and after explaining some details, this is probably best available solution,